Анимируем слоган

Funky-Juice opened the pull request:
---
:mortar_board: [Анимируем слоган](https://up.htmlacademy.ru/animation/1/user/59949/tasks/6)

:boom: https://htmlacademy-animation.github.io/59949-magic-vacation-1/5/

Funky-Juice committed:
базовый класс и метод для анимации текста

Funky-Juice committed:
метод-обработчик смены экранов

Funky-Juice committed:
метод оборачивания каждой строки текста в span элемент

Funky-Juice committed:
метод оборачивания каждой буквы текста в span элемент

Funky-Juice committed:
анимированное появление текста

Funky-Juice committed:
задействуем графический процессор на анимацию текста

Funky-Juice committed:
рандомизация времени задержки анимации

Funky-Juice committed:
отображение пробелов в тексте

Funky-Juice committed:
более плавная анимация

Funky-Juice committed:
анимация всех блоков с акцентной типографикой

Funky-Juice committed:
дополнительная опциональная задержка при создании класса анимации

Funky-Juice committed:
поправил отступы всех блоков с акцентной типографикой
(после смены строчного на строчно-блочное отображение)

Funky-Juice commented:
Привет!
Задание получилось довольно объемным (только первая часть заняла несколько дней).
Пока буду ковырять вторую часть, можешь проверить то что уже сделано.
Как проверишь - оставь коммент и я обновлю PR.

gkarev commented:
Пока полёт нормальный, отправляю на доработку в любом случае, так как есть flow работы с проектами и проверками, и нет смысла разбивать на доп. этапы, так как проверок есть несколько, сделал - отправил, получил фидбек)

Funky-Juice committed:
экран "победа-чемодан": заменил статичное изображение на составное

Funky-Juice committed:
экран "победа-чемодан": анимированное появление составного изображения

Funky-Juice committed:
экран "победа-чемодан": зацикленная анимация с фламинго

Funky-Juice committed:
экран "победа-чемодан": адаптация составного изображения под мобилки

Funky-Juice committed:
экран "победа-чемодан": подключение графического процессора для анимации

Funky-Juice committed:
экран "победа-чемодан": хак с таймаутом для отработки свойства transition
(https://stackoverflow.com/questions/54344996/css-transition-doesnt-work-if-element-start-hidden)

Funky-Juice committed:
экран "победа": доделал анимацию появления форм

Funky-Juice commented:
что-то ты быстро задание проверил...
2 минуты между взятием в работу и мёржэм ))

gkarev commented:
> что-то ты быстро задание проверил...
> 2 минуты между взятием в работу и мёржэм ))

Код я посмотрел у тебя в гитхаб, сразу, как ты только залил, благо ты толково коммиты делаешь, а "взял в работу" это статус, когда я смотрю просто визуально работу. У тебя все ок.
